impl_core/src/main/java/io/opencensus/implcore/stats/MutableViewData.java

```diff
@@ -180,6 +182,18 @@ ViewData toViewData(Timestamp now, State state) {
       }
     }
 
+    /**
+     * This method attemps to migrate this view into a reasonable state in the event of time going
+     * backwards.
+     */
+    private void handleTimeRewinds(Timestamp now) {
+      if (now.compareTo(start) < 0) {
+        // Time went backwards, physics is broken, forget what we know.
+        tagValueAggregationMap.clear();
```

Comment:
I also see `clearStats()` in this class, might be more clear

Reply:
done.